spec/services/idv/aamva_state_maintenance_window_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe Idv::AamvaStateMaintenanceWindow do
+  let(:tz) { 'America/New_York' }
+  let(:eastern_time) { ActiveSupport::TimeZone[tz] }
+
+  before do
+    travel_to Date.new(2024, 6, 2)
```

Comment:
`Date` class doesn't have a built-in timezone, also `Date` is tough to compare with `Time` because it doesn't have an hours/minutes, I would make this some sort of time
```suggestion
    travel_to eastern_time.parse('2024-06-02T00:00:00')
```